scripts/changelog.js

```diff
@@ -161,6 +168,11 @@ function isPrerelease(v) {
 	return v.includes("-");
 }
 
+// Given a version of `x.y.z-abc.n`, returns `abc`, i.e. the type of pre-release
+function prereleaseType(v) {
```

Comment:
Instead of putting this into the pre-release template, should we have a pre-template and an rc-template that inherit from the pre-release template? I know this is just a script that isn't part of the product, but all of our checking for which template to use happens outside of the templates, and having something that is switching inside the template is different and feels a little hacky rather than just doing the same way the other bits do.

Reply:
I'll do that if/when the wording for both templates differ. Until then, that's good enough honestly. This script doesn't need the same level of details as the rest of our project, it's simply a way to save time when releasing, so any shortcut in doing so is fine.
As long as I'm making releases, I'm bound to be the only one maintaining and using it 😁

Reply:
I know this isn't our main project and it's just a helper script but ... having an rc template that inherits from the pre-release one (my initial idea of 2 inheriting from the pre-release one was silly... pre is the pre-release one, so you'd only need an extra rc one) isn't a lot of work and does make it a bit nicer, IMO.

But sure, if you don't want to do it, that's fine.

Reply:
> having an rc template that inherits from the pre-release one

I can do that, yep. I'm not sure what you mean by inheriting a template, mind giving a quick example?

Reply:
Hmmm...I hadn't looked at what the template actually was, my bad. It would require quite a bit of refactoring with how it's done, so let's just leave it.

Reply:
Happy to improve whenever, let me know if you have ideas/suggestions.